components/drops/useDrops.ts

```diff
@@ -113,7 +113,7 @@ const getLocalDropStatus = (drop: Omit<Drop, 'status'>): DropStatus => {
     return DropStatus.MINTING_LIVE
   }
 
-  if (now.valueOf() - drop.dropStartTime.valueOf() <= ONE_DAYH_IN_MS) {
+  if (drop.dropStartTime.valueOf() - now.valueOf() <= ONE_DAYH_IN_MS) {
     return DropStatus.SCHEDULED_SOON
   }
```

Comment:
Why? Is it correct to change the order?

Reply:
When the execution reaches L116, `now` is always less than `dropStartTime` (due to L109), resulting in the condition on L116 being consistently true. this is clearly a bug

https://github.com/kodadot/nft-gallery/blob/57104d71eb8d8eb9008f19f2ac9995b0487d51e6/components/drops/useDrops.ts#L108-L121